.github/scripts/update-version-file.sh

```diff
@@ -12,10 +12,9 @@ if [[ "$(curl -SsL https://api.github.com/repos/anchore/${BIN}/releases/tags/${V
    exit 0
 fi
 
-echo "creating and publishing version file"
+echo "creating and publishing version file (${VERSION})"
```

Comment:
should this be `VERSION_FILE`?

Reply:
this is trying to hint what the version value will be, not necessarily where it will write it to. But I see based on the phrasing why you said that...